Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -39,15 +41,16 @@ def homebrew_extra_paths
   end
 
   def determine_extra_rpath_paths
-    paths = ["#{HOMEBREW_PREFIX}/lib"]
+    paths = ["/data/data/com.termux/files/usr/lib"]
```

Comment:
Add to https://github.com/Linuxbrew/brew/wiki/Android
```sh
ln -s /data/data/com.termux/files/usr/lib/* ~/.linuxbrew/lib/
```

Reply:
Done.